entities/built_in_entities.coffee

```diff
@@ -7,7 +7,42 @@ class BuiltInEntities
         params: [value: token: 'str']
         type: Type.FUNC
       type: Type.FUNC
-  @OutCode: (args) ->  # Make this a property of out object above
+
+    times:
+      value:
+        params: [
+          { value: token: Type.FUNC }
```

Comment:
If anyone knows a more coffeescript-y way of representing these objects inside the params array, please feel free to share.